source/MaterialXTest/CMakeLists.txt

```diff
@@ -77,6 +87,7 @@ if(MATERIALX_BUILD_RENDER) # MATERIALX_BUILD_GEN is implied; can assume relevant
   endif()
   if (MATERIALX_BUILD_OIIO)
     target_link_libraries(MaterialXTest ${OPENIMAGEIO_LIBRARIES})
+    target_compile_definitions(MaterialXTest PRIVATE MATERIALX_USE_OIIO)
```

Comment:
CMake variables are invisible to the compiler. This adds the compiler definition needed to enable the feature.

Reply:
It seems it would be simple if "build" also meant "use" then if it's set at the top level then you don't need to set it here again. We can rename the OIIO option + definition to MATERIALX_USE_OIIO if you like. Then any new modules don't need to explicitly add setting a new definition.

Reply:
Makes sense. Uploaded fix.